katversion/build.py

```diff
@@ -19,12 +19,13 @@
 import sys
 import os
 import warnings
-from distutils.command.build_py import build_py as DistUtilsBuildPy
 # Ensure we override the correct sdist as setuptools monkey-patches distutils
```

Comment:
This comment probably needs to mention `build_py` too.

Reply:
Done.